src/rpc/blockchain.cpp

```diff
@@ -1162,6 +1163,7 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
             "        }\n"
             "     }\n"
             "  }\n"
+            "  \"warnings\" : \"...\",         (string) any network and blockchain warnings.\n"
```

Comment:
nit: I think here is one whitespace missing between text (`warning[...]`and description (`(string)[...]`)